src/tools/rbd_mirror/group_replayer/Replayer.cc

```diff
@@ -346,26 +346,61 @@ void Replayer<I>::load_local_group_snapshots() {
     }
   }
 
-  //FIXME: This is not accessed under a lock
+
+  std::unique_lock locker{m_lock};
```

Comment:
This can just use the lock taken on line 330 instead by removing the braces.

Reply:
Ack.